src/containers/ToDoList/ToDoList.jsx

```diff
@@ -31,8 +33,30 @@ export const ToDoList = () => {
     setTasks(tasks.filter((task) => task.id !== id));
   };
 
+  const handleAddNewTask = () => {
+    if (newTask.trim() !== "") {
+      const newTaskObject = {
+        id: tasks[tasks.length - 1].id + 1,
```

Comment:
This won't ensure a unique IDs. If these tasks would ever come from server, they'd never be an ordered list of numbers from 1 to...

Try using the [uuid](https://www.npmjs.com/package/uuid) library to generate an unique ID.